src/content/getting-started/management-cluster/index.md

```diff
@@ -0,0 +1,119 @@
+
+## Step 3: View resources on the management cluster
+
+Let's run some commands and understand what's happening under the hood.
+
+The first command is `kubectl get nodes`. This will show us the nodes making up our management cluster.
+
+```bash
+$ kubectl get nodes
+NAME                                          STATUS   ROLES                  AGE   VERSION
+ip-10-0-5-103.eu-central-1.compute.internal   Ready    worker                 19h   v1.24.10
+ip-10-0-5-121.eu-central-1.compute.internal   Ready    control-plane,master   19h   v1.24.10
+ip-10-0-5-14.eu-central-1.compute.internal    Ready    control-plane,master   19h   v1.24.10
+ip-10-0-5-163.eu-central-1.compute.internal   Ready    control-plane,master   19h   v1.24.10
+ip-10-0-5-183.eu-central-1.compute.internal   Ready    worker                 19h   v1.24.10
+ip-10-0-5-5.eu-central-1.compute.internal     Ready    worker                 19h   v1.24.10
+ip-10-0-5-53.eu-central-1.compute.internal    Ready    worker                 19h   v1.24.10
+```
+
+There are two kinds of roles: `control-plane` and `worker`. Note that this still relates to the management cluster. None of your business applications should ever run on the management cluster's nodes, unless you want to run custom operators.
+
+The second command we suggest running is `kubectl get orgs`. This will show the organizations that are defined in your management cluster.
+
+```bash
+$ kubectl get orgs
+NAME                    AGE
+production              97d
+testing                 97d
+giantswarm              97d
+```
+
+Organizations are a way to isolate clusters, apps, etc. between different teams or environments. More information can be found in [Organizations]({{< relref "/platform-overview/organizations" >}}).
+
+Finally, run the `kubectl gs get clusters -A` command, which will show you all the clusters managed by your management cluster. The `-A` flag stands for _all namespaces_.
+
+```bash
+$ kubectl gs get clusters -A
+NAMESPACE          NAME    AGE    CONDITION   RELEASE   ORGANIZATION
+org-testing        rfjh2   84d    UPDATED     18.4.0    testing
+org-production     jn88t   91d    CREATED     18.0.1    production
+```
+
+You may notice some important points:
+
+1. A cluster belonging to an organization called `x` will be represented in the Kubernetes namespace `org-x` in the management cluster (that's why we use the `-A` flag to see all the namespaces)
+2. Organizations may not have any clusters attached to them (yet)
+3. There is a `giantswarm` organization in which we define the management cluster and its own configuration. Customers are not supported to make changes in here.
+
+Finally, you can see the YAML definition of a cluster (in this example, cluster `rfjh2` in organization `testing`) by running:
+
+```bash
+kubectl gs get cluster rfjh2 -n org-testing -o yaml
```

Comment:
why `kubectl gs get cluster` instead of just `kubectl get cluster` ? The output seems the same.

Reply:
done